config/routes.rb

```diff
@@ -17,6 +17,7 @@
   get 'splash', to: 'splash#index'
   get 'status', to: 'status#index'
   devise_for :users
+  mount Qa::Engine => '/authorities'
```

Comment:
Looks like it was in the sufia upgrade generator, so we never ran it.  https://github.com/projecthydra-labs/hyrax/commit/75f94af30aa228d4adf9878d3f144d232a8dcfb9